docs/maps/maps-getting-started.asciidoc

```diff
@@ -136,7 +136,8 @@ grids with less bytes transferred.
 ** **Visibility** to the range [0, 9]
 ** **Opacity** to 100%
 . In **Metrics**, use:
-** **Agregation** set to **Count**, and
+** **Agregation** set to **Count**,
```

Comment:
This was a little hard to follow.  Does this work:

In Metrics:

- Set **Aggreation** to **Count**.
- Click **Add metric**.
- Set **Aggregation** to **Sum** with **Field** set to **bytes**.